bulletins-et-memoires-de-la-societe-d-anthropologie-de-paris-en.csl

```diff
@@ -0,0 +1,571 @@
+                    <date-part prefix=" " name="day" range-delimiter=" au "/>
+                    <date-part prefix=" " name="month"/>
+                  </date>
+                </if>
+              </choose>
+            </group>
+          </if>
+          <else>
+            <group prefix="(" suffix=")">
+              <text term="no date" form="short"/>
+              <text variable="year-suffix" prefix="-"/>
+            </group>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="issued-sort">
+    <choose>
+      <if type="article-journal bill book chapter graphic legal_case legislation motion_picture paper-conference report song" match="none">
+        <date variable="issued">
+          <date-part name="year"/>
+          <date-part name="month"/>
+          <date-part name="day"/>
+        </date>
+      </if>
+      <else>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued-year">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+        <text variable="year-suffix"/>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+        <text variable="year-suffix" prefix="-"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal article-magazine" match="any">
+        <group prefix="" delimiter="">
+          <choose>
+            <if variable="issue">
+              <group>
+                <text variable="volume" font-style="normal"/>
+                <text variable="issue" prefix="(" suffix=")"/>
+                <text variable="page" prefix=":"/>
+              </group>
+            </if>
+            <else>
+              <text variable="volume" font-style="normal" suffix=":"/>
+              <text variable="page"/>
+            </else>
+          </choose>
+        </group>
+      </if>
+      <else-if type="article-newspaper">
+        <group delimiter=" " prefix=", ">
+          <text variable="page"/>
+        </group>
+      </else-if>
+      <else-if type="book chapter graphic motion_picture paper-conference song" match="any">
+        <group prefix=" (" suffix=")" delimiter="; ">
+          <group delimiter=", ">
+            <group>
+              <text term="volume" form="short" plural="true" suffix=" "/>
+              <number variable="number-of-volumes" form="numeric" prefix="1-"/>
+            </group>
+            <group>
+              <text term="volume" form="short" suffix=" "/>
```

Comment:
set the suffix as a delimiter on the group. Please revise the styles throughout for this issue